lib/config.js

```diff
@@ -13,33 +13,33 @@ let COFFEE_SCRIPT_AVAILABLE = false
 let LIVE_SCRIPT_AVAILABLE = false
 let TYPE_SCRIPT_AVAILABLE = false
 
-// Coffee is required here to enable config files written in coffee-script.
+// Coffee is required here to enable config files written in coffeescript.
 // It's not directly used in this file.
 try {
-  require('coffee-script').register()
+  require('coffeescript').register()
   COFFEE_SCRIPT_AVAILABLE = true
-} catch (e) {}
+} catch (e) { }
 
 // CoffeeScript lost the hyphen in the module name a long time ago, all new version are named this:
 try {
   require('coffeescript').register()
   COFFEE_SCRIPT_AVAILABLE = true
-} catch (e) {}
+} catch (e) { }
 
 // LiveScript is required here to enable config files written in LiveScript.
 // It's not directly used in this file.
 try {
   require('LiveScript')
   LIVE_SCRIPT_AVAILABLE = true
-} catch (e) {}
+} catch (e) { }
 
 try {
   require('ts-node').register()
   TYPE_SCRIPT_AVAILABLE = true
-} catch (e) {}
+} catch (e) { }
 
 class Pattern {
-  constructor (pattern, served, included, watched, nocache, type) {
+  constructor(pattern, served, included, watched, nocache, type) {
```

Comment:
Please don't change code styling. There should be space between `constructor` (or function name in other cases) and `(` (here and also other places). 

I'll try to add eslint rule for that in future, but for now please fix it manually.